src/graphql/main/schema.graphql

```diff
@@ -1172,9 +1182,9 @@ type Transaction {
   settlementFee: SignedAmount!
 
   """
-  Price in USDCENT/SETTLEMENTUNIT at time of settlement.
+  Price in WALLETCURRENCY/SETTLEMENTUNIT at time of settlement.
   """
-  settlementPrice: Price!
+  settlementPrice: SettlementPriceInMinorUnit!
```

Comment:
Tongue-in-cheek suggestion, would `PriceOfOneSettlementMinorUnitInDisplayMinorUnit` be too long for here?

Reply:
Yeah I think its a good suggestion - cannot think of anything better really.

Reply:
Updated https://github.com/GaloyMoney/galoy/pull/2439/commits/1dfb099095a98c412a0e220e2342ab0ccf04b6a3